### PR TITLE
gigasecond exercise

### PR DESCRIPTION
**Pull request by tkerdonc**

### CODE REVIEW

File: gigasecond/gigasecond.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+package gigasecond
+
+import "time"
+
+// Adds a gigasecond to an input time
+// parameters
+// * inputTime : time.Time structure to which this function will add a
+//				 Gigasecond
+func AddGigasecond(inputTime time.Time) time.Time {
+	gigasecondDuration := time.Duration(1000000000) * time.Second
```
Comment (ivajloip) [inline gigasecond/gigasecond.go]: nit: `time.Duration(1000000000)` could be extracted in a constant `gigaSecond` maybe.

File: gigasecond/gigasecond.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// This is a "stub" file.  It's a little start on your solution.
+// It's not a complete solution though; you have to write some code.
+
+// Exercism package provinding a function adding a gigasecond to an
+// input date
+package gigasecond
+
+import "time"
+
+// Adds a gigasecond to an input time
```
Comment (ivajloip) [inline gigasecond/gigasecond.go]: You are violating the recommendations for writing comments [here][1] by not starting with the method name first and not ending with a dot.

[1]: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

**Review by ivajloip — COMMENTED** [review]
A couple of minor comments, otherwise it's good :slightly_smiling_face:

File: gigasecond/gigasecond.go (hunk 2 above)
Comment (tkerdonc) [reply gigasecond/gigasecond.go]: Did no know about that, will keep in mind.

File: gigasecond/gigasecond.go (hunk 1 above)
Comment (tkerdonc) [reply gigasecond/gigasecond.go]: Thanks ! Do we have any code practice regarding constants naming/case ? e.g. all uppercase as in the last push ?

**Pull request merged**